features/provider/finance/invoices.feature

```diff
@@ -0,0 +1,43 @@
+@ignore-backend @javascript
+Feature: Show all provider's invoices
+  In order to manage invoices
+  As a provider
+  I want to be able to see all invoices issued by me
+
+  Scenario: Display years of invoices when there are invoices
+    Given a provider is logged in on 1st January 2011
+    And a buyer "zoidberg" signed up to provider "foo.3scale.localhost"
+    And the provider is charging its buyers in prepaid mode
+    And an invoice of buyer "zoidberg" for January, 2011
+    And an invoice of buyer "zoidberg" for January, 2014 on 1st January 2014 (without jobs)
+    When I go to the invoices issued by me
+    Then the years selector should have the following options:
+      |      |
+      | 2014 |
```

Comment:
Better parse all table elements so that we don't have a header element. 
```
| 2014 |
| 2013 |
| 2012 |
| 2011 |
```

But while on it, why don't we only see `2014` and `2011`?

Reply:
It is not a header, it is an empty element, which is what is seen in the UI:

![Screenshot from 2023-09-12 14-46-50](https://github.com/3scale/porta/assets/1270328/ef93bbd2-1109-44e4-9ec9-b2a1edbf186c)

Yes, having only `2011` and `2014` would be more logical, but this is how this [select_year](https://github.com/3scale/porta/blob/THREESCALE-9592-invoice-years/app/views/finance/provider/invoices/index.html.erb#L27) helper works, see https://github.com/rails/rails/blob/v6.0.6.1/actionview/lib/action_view/helpers/date_helper.rb#L638-L665